requirements.txt

```diff
@@ -0,0 +1,4 @@
+aiofile==1.5.2
+argparse==1.4.0
+aiohttp==3.6.2
+aiodns==2.0.0
```

Comment:
You're not using this, right?

Reply:
I just follow the advice here: https://aiohttp.readthedocs.io/en/stable/